app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: :User, foreign_key: :author_id
+  has_many :comments, foreign_key: :post_id
+  has_many :likes, foreign_key: :user_id
```

Comment:
- [ ] @simplegoose Great work adding associations in your post model, however in this `has_many` relationship you are using the wrong foreign key.`Also, you don't need to use foreign keys for the relationship to work you can just shorten your code by removing all foreign keys in this model, this also applies to the like and comment models `.